src/main/java/racingcar/domain/Race.java

```diff
@@ -13,31 +14,47 @@ public Race(List<Car> cars) {
         this.cars = cars;
     }
 
-    public void repeatGame(int times) {
-        for (int j = 0; j < times; j++) {
-            for (Car car : cars) {
-                compareWithFour(car, (int) Math.floor(Math.random() * MAX_NUMBER));
-            }
-            OutputView.printCurrentPosition(cars);
-            System.out.println();
+    public void repeatGame(Times times) {
+        int count = times.getTimes();
+        for (int j = 0; j < count; j++) {
+            raceOneStage(cars);
         }
```

Comment:
아래와 같이 times 객체의 상태 값을 외부애 꺼내서 직접 로직을 수행하는 것이 아니라 메시지를 보내보면 어떨까요? 🤗
[getter를 사용하는 대신 객체에 메시지를 보내자(https://tecoble.techcourse.co.kr/post/2020-04-28-ask-instead-of-getter/)

```suggestion
        while(times.isOngoing()) {
            for (int j = 0; j < count; j++) {
                raceOneStage(cars);
            }
            times.decrease;
        }
```